addons/event/migrations/11.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,87 @@
+---Fields in module 'event'---
+event        / event.event              / auto_confirm (boolean)        : not a function anymore
+# Nothing to do
+
+event        / event.event              / date_begin_located (datetime) : type is now 'char' ('datetime')
+# Nothing to do
+
+event        / event.event              / date_end_located (datetime)   : type is now 'char' ('datetime')
+# Nothing to do
+
+event        / event.event              / is_online (boolean)           : NEW
+# Nothing to do
+
+event        / event.event              / reply_to (char)               : DEL
+# Nothing to do
+
+event        / event.event              / twitter_hashtag (char)        : previously in module website_event
+# Nothing to do
```

Comment:
You have to use openupgradelib method: https://github.com/OCA/openupgradelib/blob/aab221abfcf7100402942a935ef9c9cc34e01a94/openupgradelib/openupgrade.py#L1885